src/Login/Login.jsx

```diff
@@ -93,29 +48,33 @@ class Login extends React.Component {
       next = basename === '/' ? queryParams.next : basename + queryParams.next;
     }
     return (
-      <LoginFrame>
-        <LeftBox style={{ height: `${this.state.height}px` }}>
+      <div className="login-page">
+        <div className="login-page__side-box">
```

Comment:
do we need the `height` that was being passed to `LeftBox`?

Reply:
`height` is not used, so removed.